ompi/tools/mpirun/main.c

```diff
@@ -188,30 +244,7 @@ int main(int argc, char *argv[])
     ret = execv(full_prterun_path, prterun_args);
     opal_show_help("help-mpirun.txt", "prterun-exec-failed",
                    1, full_prterun_path, strerror(errno));
-    exit(1);
-}
+    exit(1);       
+}   
+#endif /* OMPI_HAVE_PRTE_LAUNCH */
 
-/*
```

Comment:
I don't think you want to remove these copyrghts.

I think they're at the bottom of the file because MPI-enabled debuggers would default to opening the source code for mpirun.c upon attach, so we wanted to show the specific banner at the top of this file (vs. all the copyrights).  But we still need the copyrights.

Reply:
oh i see.  didn't realize that.